{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/config.py

```diff
@@ -34,9 +43,14 @@ class TestProfile(object):
     KEG_KEYRING_ENABLE = False
 
     # Mail related tests need to have this set, even though actual email is not generated.
-    MAIL_DEFAULT_SENDER = '{{cookiecutter.developer_email}}'
+    MAIL_DEFAULT_SENDER = 'test@test.com'
```

Comment:
Maybe change to "devteam+i-better-not-get-email-from-these-tests@level12.io" so that in the off-chance that someone actually starts sending emails during testing we at know about it?